src/app/perspective-status.component.ts

```diff
@@ -227,16 +266,16 @@ export class PerspectiveStatus implements OnChanges {
 
     // If nothing has changed, return an empty animation.
     if (hide === this.shouldHideStatusWidget) {
-      console.debug('Returning without update status widget visibility animation');
+      console.debug('Returning without update status widget visibility animation.');
       return new TimelineMax({});
     } else {
-      console.debug('Getting update status widget visibility animation');
+      console.debug('Getting update status widget visibility animation.');
     }
 
+    this.isPlayingShowOrHideLoadingWidgetAnimation = true;
```

Comment:
There are different types of loading animations that can play, and I want to keep track of them because things that are allowed (visibility in templates being one) are different depending on these states. We also have a general isLoading and another flag for showing or hiding text details. I prefer this being more descriptive for that reason, because there are other types of animations we play other than show/hide (for example, pulse, fade in/out).